File: src/main/java/entities/gestionuser/AbonnementDetails.java
package entities.gestionuser;

public class AbonnementDetails {
    private String type;
    private double prix;

    public AbonnementDetails() {
    }

    public AbonnementDetails(String type, double prix) {
        this.type = type;
        this.prix = prix;
    }

    public String getType() {
        return type;
    }

    public double getPrix() {
        return prix;
    }

    public void setType(String type) {
        this.type = type;
    }

    public void setPrix(double prix) {
        this.prix = prix;
    }
}

File: webapp/src/GymPlus/src/Entity/AbonnementDetails.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use App\Repository\AbonnementDetailsRepository;

#[ORM\Entity (repositoryClass: AbonnementDetailsRepository::class)]
class AbonnementDetails
{

    #[ORM\Id]
    #[ORM\Column]
    private ?string $name;

    #[ORM\Column]
    private ?float $prix;

    public function getName(): ?string
    {
        return $this->name;
    }

    public function getPrix(): ?float
    {
        return $this->prix;
    }

    public function setPrix(float $prix): static
    {
        $this->prix = $prix;

        return $this;
    }


}
